repo/packages/A/artifactory/3/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "packagingVersion": "4.0",
+  "name": "artifactory",
+  "version": "5.11.1",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/JfrogDev/artifactory-dcos.git",
+  "maintainer": "soleng-team@jfrog.com",
+  "website": "https://jfrog.com",
+  "framework": false,
+  "description": "As the first, and only, universal Artifact Repository Manager on the market, JFrog Artifactory fully supports software packages created by any language or technology. Documentation is available at: https://github.com/dcos/examples/",
+  "tags": [
+    "framework",
+    "artifacts",
+    "artifactory",
+    "repository"
+  ],
+  "preInstallNotes": "You will need at least 1 public agent in your cluster to install Artifactory-lb.",
+  "postInstallNotes": "Artifactory has been installed. Please install Artifactory-lb if not already installed.",
+  "postUninstallNotes": "Artifactory has been uninstalled.",
+  "licenses": [
+    {
+      "name": "Apache License Version 2.0",
+      "url": "https://github.com/JfrogDev/artifactory-dcos/blob/master/LICENSE"
+    }
+  ],
+  "selected": true
```

Comment:
This shouldn't be a selected package or?

Reply:
We wanted this package to be selected on search. My understanding based on the package schema: This package will be picked or selected in search results.

Reply:
This determines whether packages are displayed as selected aka certified which this package is not (yet ;) ).